src/scss/components/_inputs.scss

```
@@ -144,6 +144,30 @@ $input-select-margin-right: 1;
   }
 }
 
+.usa-checkbox__label-illustrated,
+.usa-radio__label-illustrated {
```

There should be two dashes for the modifier, since we follow [BEM two-dashes style](https://en.bem.info/methodology/naming-convention/#two-dashes-style).

```suggestion
.usa-checkbox__label--illustrated,
.usa-radio__label--illustrated {
```